cluster/gce/gci/health-monitor.sh

```diff
@@ -24,11 +24,22 @@ set -o pipefail
 
 # We simply kill the process when there is a failure. Another systemd service will
 # automatically restart the process.
-function docker_monitoring {
+function container_runtime_monitoring {
+  # Container runtime startup takes time. Make initial attempts before starting
+  # killing the container runtime.
+  local -r initial_attempts=5
+  local attempt=1
+  local -r crictl="${KUBE_HOME}/bin/crictl"
+  local -r container_runtime="${CONTAINER_RUNTIME_NAME:-"docker"}"
+  until timeout timeout 60 "${crictl}" pods > /dev/null || (( attempt == initial_attempts ))
+  do
+    echo "$attempt initial attempt \"${crictl} pods\"! Trying again in $attempt seconds..."
+    sleep $(( attempt++ ))
+  done
   while [ 1 ]; do
```

Comment:
Not really something you touched... But can you please change this to `while : ; do` ? Using `[ 1 ]` is just too ugly!

Reply:
Done.